docs/framework/configure-apps/file-schema/runtime/loadfromremotesources-element.md

```diff
@@ -25,75 +25,94 @@ Specifies whether assemblies from remote sources should be granted full trust.
+
 > [!IMPORTANT]
->  If the assembly should not run in full trust, do not set this configuration element. Instead, create a sandboxed <xref:System.AppDomain> in which to load the assembly.  
-  
-## Configuration File  
- This element is typically used in the application configuration file, but can be used in other configuration files depending upon the context. For more information, see the article [More Implicit Uses of CAS Policy: loadFromRemoteSources](http://go.microsoft.com/fwlink/p/?LinkId=266839) in the .NET Security blog.  
-  
-## Example  
- The following example shows how to grant full trust to applications from remote sources.  
-  
-```xml  
+> If the assembly should not run in full trust, do not set this configuration element. Instead, create a sandboxed <xref:System.AppDomain> in which to load the assembly.
+
+The `enabled` attribute for the `<loadFromRemoteSources>` element is effective only when code access security (CAS) is disabled. By default, CAS policy is disabled in the .NET Framework 4 and later versions. If you set `enabled` to `true`, remote assemblies are granted full trust.
+
+If `enabled` is not set to `true`, a <xref:System.IO.FileLoadException> is thrown under the following conditions:
+
+- The sandboxing behavior of the current domain is different from its behavior in the .NET Framework 3.5. This requires CAS policy to be disabled, and the current domain not to be sandboxed.
+
+- The assembly being loaded is not from the `MyComputer` zone.
+
+Setting the `<loadFromRemoteSources>` element to `true` prevents this exception from being thrown. It enables you to specify that you are not relying on the common language runtime to sandbox the loaded assemblies for security, and that they can be allowed to execute in full trust.
+
+## Notes
+
+- In the .NET Framework 4.5 and later versions, assemblies on local network shares run in full trust by default; you do not have to enable the `<loadFromRemoteSources>` element.
+
+- If an application has been copied from the web, it is flagged by Windows as being a web application, even if it resides on the local computer. You can change that designation by changing its file properties, or you can use the `<loadFromRemoteSources>` element to grant the assembly full trust. As an alternative, you can use the <xref:System.Reflection.Assembly.UnsafeLoadFrom%2A> method to load a local assembly that the operating system has flagged as having been loaded from the web.
+
+- You may get a <xref:System.IO.FileLoadException> in a Windows Virtual PC application when you try to load a file from linked folders on the hosting computer. This error may also occur when you try to load a file from a folder linked over [Remote Desktop Services](http://go.microsoft.com/fwlink/?LinkId=182775) (Terminal Services). To avoid the exception, set `enabled` to `true`.
```

Comment:
This sentence reads a bit awkward to me. It's not a *Windows Virtual PC application*, but rather an *application that is running in a Windows Virtual PC*. Maybe it should be two sentences?